chartmuseum2oci/main.go

```diff
@@ -275,7 +283,13 @@ func pullChartFromChartmuseum(helmChart HelmChart) error {
 
 func pushChartToOCI(helmChart HelmChart) error {
 	repoURL := fmt.Sprintf("oci://%s/%s%s", harborHost, helmChart.Project, destPath)
-	cmd := exec.Command(helmBinaryPath, "push", helmChart.ChartFileName(), repoURL) //nolint:gosec
+	params := []string{"push", helmChart.ChartFileName(), repoURL}
+	if helmInsecure {
+		params = append(params, "--insecure-skip-tls-verify=false")
```

Comment:
You might be wondering why it's false here, but the fact is that it's quite strange.

<img width="1427" height="347" alt="Image" src="https://github.com/user-attachments/assets/75ee0e06-c12a-4013-831d-0c110ef4e3ea" />